pkg/config/default.go

```diff
@@ -562,3 +564,14 @@ func (c *Config) RootlessNetworking() string {
 func (c *Config) MachineImage() string {
 	return c.Engine.MachineImage
 }
+
+// ImageCopyTmpDir default storage tmpdir
+func (c *Config) ImageCopyTmpDir() string {
+	if path, found := os.LookupEnv("TMPDIR"); found {
+		return path
+	}
+	if c.Engine.ImageCopyTmpDir == "storage" {
```

Comment:
I’m not too fond of magic values / in-band signaling in general.

A brute-force approach would be to have an `image_copy_tmp_dir` variable with path, and an `image_copy_tmp_dir_from_storage` boolean (and to fail if both are set?).

In this case, how about enforcing that the value is either the `storage` special case, or starts with `/` to be an absolute path? Requiring an absolute path is probably a good idea anyway, and it would allow us to reasonably cleanly partition the string space between paths and special cases.

Reply:
I would rather keep this simple on the user, I now return an error if the path does not start with a "/"